operations/toc-templates/template-dd-pr-graduation.md

```diff
@@ -204,11 +204,10 @@ N/A
 
 ## Security
 
```

Comment:
Note: this section may be augmented by a joint-assessment performed by TAG Security and Compliance if completed as a suggested item prior to application.